zebra/zebra_vrf.c

```diff
@@ -541,11 +576,22 @@ int zebra_vrf_netns_handler_create(struct vty *vty, struct vrf *vrf,
 	return CMD_SUCCESS;
 }
 
+static struct cmd_node vrf_node = {
+	.name = "vrf",
+	.node = VRF_NODE,
+	.parent_node = CONFIG_NODE,
+	.prompt = "%s(config-vrf)# ",
+};
+
 /* Zebra VRF initialization. */
 void zebra_vrf_init(void)
 {
 	vrf_init(zebra_vrf_new, zebra_vrf_enable, zebra_vrf_disable,
 		 zebra_vrf_delete);
 
 	hook_register(zserv_client_close, release_daemon_table_chunks);
+	install_node(&vrf_node);
+	install_default(VRF_NODE);
```

Comment:
I used this because without I have zebra down during init with this message:
`mpls_fec_nexthop_resolution_vrf_cmd[[no$no] mpls fec nexthop-resolution]:
        node 21 does not exist.
        please call install_node() before install_element()
`
If you want to remove it you might have a comment how to overturn the consequence?